.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I'd rather use `prettier` to explicitly format code than eslint to enforce a code style.